website/docs/script/syscalls_for_script.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: syscalls-for-script
+title: "CKB Syscalls for Script"
+---
+
+# CKB Syscalls for Script
+
+A collection of CKB VM syscalls. Also include relevant constant, such as return codes, sources, cell fields, header fields, and input fields.
```

Comment:
Can you change all "CKB VM" to "CKB-VM", "script" to "Script", "cell" to "Cell", "lock script" to "Lock Script" and "type script" to "Type Script"?
Everything else looks good to me!